tools/plugins/spiffe/plugin.go

```diff
@@ -0,0 +1,84 @@
+package spiffe
```

Comment:
could you make this plugin exec'ed ? So his package would become a main and we can leverage the client-go code to call it.